Gemfile.lock

```diff
@@ -0,0 +1,109 @@
+PATH
```

Comment:
I don't think we need to commit this file

Reply:
I had some issues w/o it with the build, but will try again